implementations/rust/ockam/ockam_vault/src/signer_impl.rs

```diff
@@ -2,12 +2,16 @@ use crate::software_vault::SoftwareVault;
 use crate::xeddsa::XEddsaSigner;
 use crate::VaultError;
 use arrayref::array_ref;
-use ockam_vault_core::{Secret, SecretType, Signer, CURVE25519_SECRET_LENGTH};
+use ockam_vault_core::{Buffer, Secret, SecretType, Signer, CURVE25519_SECRET_LENGTH};
 use rand::{thread_rng, RngCore};
+use signature_bls::SecretKey;
+use signature_bbs_plus::{Issuer, MessageGenerators};
+use signature_core::lib::*;
```

Comment:
Let's avoid importing the whole module

Reply:
Sure. Now I have just imported Message type.